bazel/MODULE.bazel

```
@@ -1,28 +1,57 @@
-module(name = "envoy_toolshed", version = "")
+module(
+    name = "envoy_toolshed",
+    version = "0.3.8-dev",
```

```suggestion
    version = "0.3.11-dev",
```

atm i just manually set the release version when publishing - but perhaps we can have it read this